brambling/templates/brambling/_event_list_item.html

```diff
@@ -1,94 +1,71 @@
-{% load daguerre %}
+{% load daguerre zenaida %}
 
 {% if event.banner_image %}
 	{% adjust event.banner_image 'fill' width=940 height=300 as banner_image %}
 {% endif %}
 
+{% url 'brambling_event_shop' event_slug=event.slug organization_slug=event.organization.slug as event_url %}
+
 <div class="margin-trailer-dbl event-listing relative">
-	<a href="{% url 'brambling_event_shop' event_slug=event.slug organization_slug=event.organization.slug %}" class="a-secret a-opacity">
+	<a href="{{ event_url }}" class="a-secret a-opacity">
 		<div class="event-listing-image"{% if banner_image %} style="background-image: url({{ banner_image }})"{% endif %}>
-			<div class="datecard">
-				<h5 class="datecard-month">{{ event.start_date|date:"M" }}</h5>
-				<p class="datecard-day">{{ event.start_date|date:"j" }}</p>
-			</div>
-
-
 		</div>
 	</a>
-	{% if is_admin %}
-	<div class="btn-group relative-right relative-top">
-		<a class="btn btn-default dropdown-toggle tipped" title='Manage Event' type="button" id="manageEvent" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true" data-container="body">
-			<i class="fa fa-cog"></i>
-			Manage
-			<span class="caret"></span>
-		</a>
-		{% include "brambling/_event_manage_dropdown.html" with classes="dropdown-menu-right" %}
+	<div class="relative-right relative-top">
+		{% if is_registered %}
+			{% if event.order and event.order.has_cart %}
+				<a href="{{ event.get_absolute_url }}" class="btn btn-default tipped" title="Complete your registration now!" data-placement="bottom">
+					<i class="fa fa-exclamation-circle"></i> Cart expires in
+
+					{% td_to_dict event.order.cart_expire_time|td_timeuntil as time_left %}
+
+					<strong data-countdown="timer">
+						{% if time_left.days %}<span data-countdown="days">{{ time_left.days }}</span> day{{ time_left.days|pluralize }}{% endif %}
+						{% if time_left.hours %}<span data-countdown="hours">{{ time_left.hours }}</span>:{% endif %}<span data-countdown="minutes">{% if time_left.minutes < 10 and time_left.hours %}0{% endif %}{{ time_left.minutes }}</span>:<span data-countdown="seconds">{% if time_left.seconds < 10 %}0{% endif %}{{ time_left.seconds }}</span>
```

Comment:
Can we clean all this up a bit? It being on one smushed line makes this hard to parse and would make future amendments difficult.